### DISCUSSION TITLE
How to account for piecework

### DISCUSSION THREAD

**Discussion by houstonbofh** [kimai/kimai#4344] [General]
Most of my work is hourly, but some is bid or piecework.  Been trying for a while, but can not find a way to enter piecework.  Or a bid project...

To be clear, this is like "Mount 50 Access Points @ $20" each or "Upgrade firewall @ $400."  All I can find is to create a new project for each, but it is fairly common so I would have a LOT of additional projects.

**Comment by kevinpapst** [comment]
For piece work the expense bundle is the right thing: https://www.kimai.org/store/expenses-bundle.html
You can test it in the demo.

Never heard of bid work. What is that?

**Reply by houstonbofh** [reply]
I bid a set price for a complete specific job.  Like $1500 to replace your core router and all routes. I am giving them security of price, but assuming risk of underestimating time.

As to the expense bundle, that is not open source, correct?  No way to do this in a fully FOSS setup?
